static/docs/commands-reference/push.md

```diff
@@ -24,9 +24,10 @@ environments and preserving data versions (input datasets, intermediate results,
 models, metrics, etc) remotely (S3, SSH, GCS, etc) are the most common use cases
 for these commands.
 
-The `dvc push` command allows one to upload data to remote storage. It is run
-along with 'git commit' and
-[git push](https://git-scm.com/docs/git-push/1.5.5.2).
+The `dvc push` command allows one to upload data to remote storage. It is 
+usually run along with `git commit` and `git push`. `dvc push` and `git push`
```

Comment:
intention is not to explain the command (git push), but to emphasize that `dvc push` is not doing anything magical, that it's not running git push for you or something. It's already done by the first two sentences to some extent. I would just modify them a little bit to be more specific:

> The `dvc push` command allows one to upload data to remote storage. It does not save changes to code or DVC-files. You still need to run `git commit` and `git push` to save them.

Something like this ^^

@jorgeorpinel could you take a look, does the phrase above look reasonable?

Reply:
```
The `dvc push` command allows one to upload data to remote storage. It doesn't
update any modified changes in the code or DVC-files. The changes made in the 
the files or addition of any new file still needs to be saved by using        
`git commit` and `git push`.
```
Is it good?

Reply:
`update any modified change` -> save any changes

"The changes made in the the files or addition of any new file still" -> Those should be saved ...